DOCUMENTATION.md

```diff
@@ -275,7 +275,7 @@ You can provide settings to a Hops application via a `"hops"` key in your `packa
 
 | Name | Type | Default | Example | Description |
 | --- | --- | --- | --- | --- |
-| `https` | `Boolean | Object` | `false` | `true` or<br/>`{ "keyFile": "./my.key", "certFile": "./my.cert" }` | Configure HTTPS support for Hops |
+| `https` | `Boolean` or `Object` | `false` | `true` or<br/>`{ "keyFile": "./my.key", "certFile": "./my.cert" }` | Configure HTTPS support for Hops |
```

Comment:
Please write it like `Boolean \| Object`, to stick with a more formal syntax.

Reply:
Done :)

Reply:
No, I meant like in a single inline code element:

```
`Boolean \| Object`
```

Reply:
Coolio.